includes/components/class-admin.php

```diff
@@ -1681,6 +1817,51 @@ public function render_notices() {
 		// Add default notices.
 		$notices = [];
 
+        // Get unlicensed products.
+        $unlicensed_products = $this->get_unlicensed_products();
+
+        if ( $unlicensed_products['themes'] ) {
+            $notices['unlicensed_themes'] = [
+                'type'        => 'warning',
+                'dismissible' => true,
```

Comment:
Please make it non-dismissible, but it should check activated theme/extensions only